File: containers/p1/python/template.py
import json

# insert Solution class here
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        pass

with open("testcase.json", "r") as read_file:
    data = json.load(read_file)

sol = Solution()
isOk = True
# for each input pair, check if Solution.twoSum() returns a matching expected answer
for i in range(len(data["input"]["nums"])):
    answer = sol.twoSum(data["input"]["num"][i], data["input"]["target"][i])
    if answer != data["expected"][i]:
        # write json to file
        isOk = False
        break

if isOk:
    # write json to file
    print("ok")

print("done")

File: containers/p1/js/template.js
const fs = require("fs")
const tc = require("../testcase.json")

// insert Solution class here
var twoSum = function(nums, target) {
    const output = []
    nums.forEach(num => {
        output.push(num + target)
    });

    return output
};

let isOk = true
for (let i = 0; i < tc.input.nums.length; i++) {
    const answer = twoSum(tc.input.nums[i], tc.input.target[i])
    if (answer != tc.expected[i]) {
        const output = {
            "result": "wrong",
            "input": tc.input.nums[i],
            "expected": tc.expected[i],
            "output": answer
        }
        const data = JSON.stringify(output)
        fs.writeFile("result.json", data, (err) => {
            if (err) {
                throw err
            }
        })
        isOk = false
    }
}

if (isOk) {
    const output = {
        "result": "OK"
    }
    const data = JSON.stringify(output)
    fs.writeFile("result.json", data, (err) => {
        if (err) {
            throw err
        }
    })
}

console.log("done")